project.clj

```diff
@@ -4,9 +4,11 @@
   :min-lein-version "2.0.0"
   :dependencies [[org.clojure/clojure "1.9.0"]
                  [compojure "1.6.1"]
+                 [ring "1.7.0-RC1"]
```

Comment:
Do we need this since ring server works? Does it not matter and it's benign?

Reply:
i may want to run from the repl at some point so let's keep it